packages/ckeditor5-engine/src/view/domconverter.js

```diff
@@ -1644,8 +1661,15 @@ function hasBlockParent( domNode, blockElements ) {
  * {@glink framework/guides/architecture/editing-engine#editing-pipeline editing pipeline} of the editor. To avoid this,
  * the `<script>` element was renamed to `<span data-ck-unsafe-element="script"></span>`.
  *
- * @error domconverter-unsafe-element-detected
- * @param {module:engine/model/element~Element|HTMLElement} unsafeElement The editing view or DOM element
```

Comment:
Why did you remove the element itself from here?

Reply:
I noticed that it was not consistent - in one place we were providing the DOM element as a context, but in the other we were giving the View element and DOM element we could provide would be already a replaced element. I decided to get rid of it altogether. 

The error description is clear enough and finding offending `<style>` or `<script>` tag in the raw document shouldn't be so hard that we have to point it out to the user.